sled-agent/src/services.rs

```diff
@@ -883,65 +863,168 @@ impl ServiceManager {
         Ok(())
     }
 
-    pub async fn ensure_switch(&self, request: Option<ServiceZoneRequest>) {
+    /// Ensures that a switch zone exists with the provided IP adddress.
+    pub async fn activate_switch(
+        &self,
+        switch_ip: Option<Ipv6Addr>,
```

Comment:
Kind of a nit, but can we rename this to `service_ip` or something like that? IIUC, this is the link-local IPv6 we use for `dpd`, MGS, and `tfportd`, and not an IP address on the switch itself.

Reply:
IMO `service_ip` may be ambiguous in the context of the service manager, and I kinda like that the name is uniform between files. WDYT about `switch_zone_ip`? (See: e42b4689823c8e4064937ec557f5da3bd9c75406)

Reply:
That's good! Just something to be clear it's for software running in the switch zone on the host, not anything particularly related to the switch itself.